client/src/components/layout/SaveTableForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react"
+import translateServerErrors from "../../services/translateServerErrors.js"
+import ErrorList from "./ErrorList.js"
+
+const SaveTableForm = ({ selectedPlayers }) => {
+  const [form, setForm] = useState({
+    title: "",
+    notes: "",
+    players: [],
```

Comment:
I see players here, what's the difference between players and selectedPlayers? doesn't look like `players` is being used. When you add to the formpayload, you set `players` as the `selectedplayers`